src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketPal.Unix.cs

```diff
@@ -321,23 +323,29 @@ private static unsafe long SendFile(SafeSocketHandle socket, SafeFileHandle file
 
```

Comment:
```suggestion
[SkipLocalsInit]
```

Reply:
Not sure how much this would affect perf of zeroing the stackalloc'ed arrays, but I suppose I can't hurt?

Reply:
Thanks, but it won't affect it at all.  We disable locals init for all of netcoreapp.